### DISCUSSION TITLE
Pair Extraordinaire achievement from private organisation repository

### DISCUSSION THREAD

**Discussion by twistezo** [community/community#31520] [Other Feature Feedback, Questions, & Ideas]
We are working inside private GitHub organisation on private repository.
We have a lot of coauthored commits on default branch **but none of them are counted on our public profiles.**

Usually we create Pull Request. Each one adds some commits. One of us in GitHub web interface click `Squash and merge` (to default branch) and GH adds to commit message info about authors.

Example of our commit from `git log`:
```
commit 123...
Author: Foo Bar <foobar@gmail.com>
Date:   Wed Aug 31 12:56:36 2022 +0200

    Some msg

    Co-authored-by: Foo Baz <foobaz@gmail.com>
```
where both e-mails are connected to public GH accounts.

Where is the problem? Private org, private repo or maybe in commit message?

PS. I have `Include Private contributions on my profile` checked in my GH options.

**Comment by airtower-luna** [comment]
From my experience the co-authored commit must exist in a PR *before* merging, and you get the achievement when that PR gets merged.

**Comment by F4NT0** [comment]
Try in a **public** repo, put your colleage as a **Collaborator** and he needs to **accept** on the email, create a **new branch**, change something and make the commit on the github web putting like the image below the **co-author message** and **create a PR**, put you college as a **reviewer** and **merge the PR without the review**, it works for me for the YOLO badge and the Pair Extraordinaire.

![image](https://user-images.githubusercontent.com/18719295/187834181-80e5b230-3ffa-47ec-aa9a-bcf5ebad2a3f.png)

**Comment by Anmol-Baranwal** [comment]
You can check my profile [here](https://github.com/Anmol-Baranwal) & you can see that my co-authored commit was counted in a private repository, of which I deleted and the link became inaccessible. 

So, yes you can get it even in a private repository but you must make a PR from a different branch, and you can just make one co-authored commit for the PR changes which will do just fine.

**Reply by Anmol-Baranwal** [reply]
@twistezo 
I just wanted to check in and see if you have any remaining questions or concerns about my answer. If not, would you please mark my response as accepted?

It would be greatly appreciated and it will also help other users who might be looking for a solution to a similar issue.
